pages/settings.js

```diff
@@ -675,6 +676,18 @@ class UserSettings extends Component {
       });
   };
 
+  deleteAccount = async () => {
+    const deleteOrNot = confirm(
+      "Are you sure you want to delete your account?"
+    );
+
+    if (deleteOrNot) {
+      const url = generateApiUrl(`user/${this.props.user.id}`);
+      const res = await fetch(url, api.DELETE_CONFIG());
```

Comment:
Do we need to log them out?

Reply:
This will auto log them out, and then we just refresh the page to the home page.